### ISSUE TITLE
Ability To Set Sleep Timer To Any Amout Of Time 

### ISSUE THREAD

**Issue by whatinthenameofgod** [Automattic/pocket-casts-android#940]
### What problem does this address?

The sleep timer arbitrarily won't work for anything below 5 minutes and only works for multiplies of 5 minutes and you can't set seconds.

### What is your proposed solution?

Keep quick time shortcuts but let people set more specific timers for any time and not just multiples of 5 minutes.

### Did you search for existing feature suggestions?

- [X] I have searched for existing feature suggestions.

**Comment by CookieyedCodes** [comment]
I think a better solution is add a plus +1 minute to this screen & add a number of episodes sleep timer as I think that adds enough granularity for more custom times, should be simple enough too do too, pity I suck at code 😭

 ![Screenshot_2023-05-06-20-05-39-29_28ad70af3b47247953fcd94176b9a9c1.jpg](https://user-images.githubusercontent.com/20846898/236642262-168e3ea9-a1d4-44e2-bb5f-d33872e247b4.jpg)

**Comment by whatinthenameofgod** [comment]
I like adding a number of episodes as a deeper and more granular option, but the +1 minute is a bandaid and still wouldn't solve the inability to set something under 5 minutes. The underlying structure of setting a timer shouldn't be arbitrarily set to a specific number or multiples thereof.

**Comment by CookieyedCodes** [comment]
Fair enough, I would probably enable it so that going less then 5 mins on the custom timer would have single digits (with maby a long press on the custom timer load up the ability to input where you could set a completely custom time then) I do think +1 min would be a quicker fix tho as the code should allready be in place & it would enable timer extension more easily 🤔🤔

**Comment by neeldoshii** [comment]
@whatinthenameofgod @CookieyedCodes from both of your conversation what if we allow custom time for user to pick.
<img width="522" alt="Screenshot 2023-09-26 at 9 01 04 PM" src="https://github.com/Automattic/pocket-casts-android/assets/60827173/f25a1822-cb0a-4bfa-b890-68e2718ab664">
I guess this could be the best approach.

**Comment by CookieyedCodes** [comment]
@neeldoshii so I was able to get someone to add a +1 minute & the < then 5 mins numbers if you have noticed, anything less then 5 mins dose seem to be a bit inaccurate tho but that I am guessing is a limitation of Android (timers can be a bit weird on android from my understanding battery management is messy with inaccurate timers unfortunately) but I'm not really haveing issues other then that 😅

Since intergrating this my sleeptimer I am mostly happy with it as my sleep timer niggles have moved to the following two niggles.
1. Number of episodes sleep timer (I have done mockups for this, which I can share if interested) &
2. A clock setting to activate a spefic sleep timer at a certain user selectable time of day (ie are you still listening like Netflix)

(To be clear not a member of the team of pocket casts so I can't give approval, I generally just think of ways the app could improve, in kind of a power user & like thinking on UX/makeing pocketcasts amazing 😅, pity I suck at code tho)

 (someone should add +1 minute & the < then 5 minutes to iOS tho 😅)
